questions.md

```diff
@@ -0,0 +1,3 @@
+* When I have to open a PR?
```

Comment:
When you've created a branch of code you would like to merge into another, usually master.